python/lsst/rubintv/production/starTracker.py

```diff
@@ -151,6 +153,30 @@ def dayObsSeqNumFromFilename(filename):
     return dayObs, seqNum
 
 
+def getFilename(camera, dayObs, seqNum):
+    """Get the filename for a given camera, dayObs and seqNum.
+
+    Parameters
+    -------
+    camera : `lsst.rubintv.production.starTracker.StarTrackerCamera`
+        The camera.
+    dayObs : `int`
+        The dayObs.
+    seqNum : `int`
+        The seqNum.
+
+    Returns
+    -------
+    filename : `str`
+        The filename.
+    """
+    rootPath = '/project/GenericCamera/'
```

Comment:
Below you have a `locationConfig`.  Should that be used here instead of hard coding the path?

Reply:
Admittedly that's not ideal, but actually the `locationConfig` only holds  the `starTrackerDataPath: '/project'` part, the `GenericCamera` part is hardcoded elsewhere, and soon all this will be butler driven once Robert gets that stuff all working, so this part in particular is very temporary, so I don't think it's worth the hassle of getting the location config in there right now.